docs/reference/developer_documentation.md

```diff
@@ -43,6 +43,14 @@ poetry run python buildingmotif/api/app.py
 ```
 API will run on localhost:5000
 
+### Using Postgres
+
+**In Development**: While we find SQLite much easier to use for development, there are reasons to use Postgres as the backend database for BuildingMOTIF during development.
+We recommend use of the `psycopg2-binary` package for interacting with Postgres for development. *This will be installed automatically as part of installing development dependencies. Make sure you use `poetry install --with dev`*.
+
+**In Prodution**: To use Postgres as the backend database in a production deployment, we recommend installation of BuildingMOTIF with the `postgres` feature (`pip install BuildingMOTIF[postgres]`).
```

Comment:
```suggestion
**In Production**: To use Postgres as the backend database in a production deployment, we recommend installation of BuildingMOTIF with the `postgres` feature (`pip install BuildingMOTIF[postgres]`).
```

Reply:
do we also want to include the `poetry add "BuildingMOTIF[postgres]"` line as well?

Reply:
Oh, as in if you wanted to add it as a dependency to another project?

Reply:
> Oh, as in if you wanted to add it as a dependency to another project?

yeah, I don't have strong feelings about it.

Reply:
Added a quick sentence -- why not? No harm in being more helpful

Reply:
@TShapinsky can you approve when you get a chance?